content/blog/lifehacking/2017-12-01_freedom_and_productivity.md

```diff
@@ -0,0 +1,199 @@
+Title: Freedom and productivity: an open mind manifesto
+Date: 2017-05-12 10:00
+Author: Benjamin Bouvier
+Tags: lifehacking
+Slug: freedom-and-productivity
+Lang: en
+
+![Freedom and productivity]({filename}/images/keep-exploring.jpg)
+
+There are many self-development articles out there on blogs, evocating all the
+techniques and methods that one can use to work better, be happier, get more
+successful, whatever you can dream of. It really feels these techniques can all
+be reduced to a well-defined reduced set of core habits. Work out. Practice
+journaling. Meditate. Be grateful. Read a lot. Drink a liter of water every
+day. Alright, you get the idea; if you're reading this you probably went
+through a whole lot of other very similar self help posts in the past. Or even
+read books about the topic. However, they can easily be misinterpreted. Let's
```

Comment:
"they" here isn't well defined. I would rephrase this so that it's clearer what it refers to. The previous sentence might be the culprit.

Reply:
Use `those` instead, maybe? Then to me it would be clearer that you refer to books and blog posts.